lib/cocoapods/command/lib/lint.rb

```diff
@@ -21,6 +21,7 @@ def self.options
              '(defaults to https://github.com/CocoaPods/Specs.git). ' \
              'Multiple sources must be comma-delimited.'],
             ['--private', 'Lint skips checks that apply only to public specs'],
+            ['--swift-version=VERSION', 'The SWIFT_VERSION that should be used to lint the spec'],
```

Comment:
Can we add: "This takes precedence over a .swift-version file containing SWIFT_VERSION."

This will include `.swift-version` in the help message, which will create the opportunity for people to be exposed to `.swift-version` before linting, since otherwise it's a message you get if the swift version isn't set.